### PR TITLE
OSOE-554: Update deprecated links in the summary of the O1 version of the Training Demo module

### PR DESCRIPTION
**Pull request by MZaXiMuS**

### CODE REVIEW

File: OrchardHUN.TrainingDemo.Tests/StartHere.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ Other gotchas:
 
 If you don't really know what unit testing is about please study the topic first.
 
-There is no scaffolding command to create unit test projects (yet: http://orchard.codeplex.com/workitem/18696) so you have to create them by hand.
+There is no scaffolding command to create unit test projects (yet: https://www.google.com/search?q=Work+Item+Codeplex) so you have to create them by hand.
```
Comment (domonkosgabor) [inline OrchardHUN.TrainingDemo.Tests/StartHere.txt]: Instead of this URL, you can link this URL that contains the https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Modules/Orchard.CodeGeneration/Commands/CodeGenerationCommands.cs#L134 the given scaffolding command. And of course, you need to rephrase this to something like:

You can use the codegen moduletests scaffolding command to create unit test projects (https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Modules/Orchard.CodeGeneration/Commands/CodeGenerationCommands.cs#L134).

And you can remove the rest of this paragraph.

File: Models/PersonListPart.cs [hunk 2]
```diff
@@ -3,8 +3,8 @@
  * by their sex.
  * It's OK to place the part and the part record into the same code file if neither of them are particularly big.
  * Make sure to read the corresponding documentation:
- * http://docs.orchardproject.net/Documentation/Creating-a-module-with-a-simple-text-editor
- * http://docs.orchardproject.net/Documentation/Writing-a-content-part
+ * http://docs.orchardcore.net/projects/O1/en/latest/Documentation/Creating-a-module-with-a-simple-text-editor
```
Comment (domonkosgabor) [inline Models/PersonListPart.cs]: I think it's better now to add these URLs with https instead of http. So, please, replace the http everywhere (not just the http://docs.orchardcore.net ones) to https.

File: Services/DynamicEventHandler.cs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿/*
  * Events are more than just a convenient way of calling into all the registered dependencies. Actually calls to event 
- * go through the so-called Orchard Event Bus (see: http://docs.orchardproject.net/Documentation/How-Orchard-works#EventBus). 
+ * go through the so-called Orchard Event Bus (see: http://docs.orchardcore.net/projects/O1/en/latest/Documentation/How-Orchard-works#EventBus).
```
Comment (domonkosgabor) [inline Services/DynamicEventHandler.cs]: The correct URL here is: https://docs.orchardcore.net/projects/O1/en/latest/Documentation/How-Orchard-works/#event-bus

File: StartHere.txt [hunk 4]
```diff
@@ -10,7 +10,7 @@ Before you dive deep into this module it'd be best if you made sure that you hav
 *	You know what MVC is about and how ASP.NET MVC works. It's important that you understand how ASP.NET MVC works or 
 	generally what MVC is about. If you are not familiar with the topic take a look at the tutorials at http://www.asp.net/mvc.
 	Or for a funnier start, watch this video: http://channel9.msdn.com/Blogs/matthijs/ASPNET-MVC-2-Basics-Introduction-by-Scott-Hanselman
```
Comment (domonkosgabor) [inline StartHere.txt]: I cannot find this video anywhere on the internet now. :( So, you could remove this line.

File: OrchardHUN.TrainingDemo.Tests/StartHere.txt [hunk 5]
```diff
@@ -13,7 +13,7 @@ Other gotchas:
```
Comment (domonkosgabor) [inline OrchardHUN.TrainingDemo.Tests/StartHere.txt:13]: Will write it here because the mentioned file is not in the PR.
The PersonManagerTests.cs contains a URL: http://code.google.com/p/moq/
Moq is now in GitHub, so, you can replace this URL with the current one: https://github.com/Moq/moq4

**Review by domonkosgabor — CHANGES_REQUESTED** [review]

File: StartHere.txt (hunk 4 above)
Comment (MZaXiMuS) [reply StartHere.txt]: Maybe this? Same presenter and similar topic :)
https://learn.microsoft.com/en-us/shows/asp-net-site-videos/5-minute-introduction-to-aspnet-mvc

Comment (domonkosgabor) [reply StartHere.txt]: Nice, this will do it! :)

File: Models/PersonListPart.cs (hunk 2 above)
Comment (MZaXiMuS) [reply Models/PersonListPart.cs]: In my practice project only 'http://localhost...' works for me, so I left it this way in Controllers\YourFirstOrchardController.cs. Should I change this also?

Comment (MZaXiMuS) [reply Models/PersonListPart.cs]: ( And owin.org (Services\OwinMiddleware.cs) does not work well with https. )

Comment (domonkosgabor) [reply Models/PersonListPart.cs]: Don't change it "everywhere" just where it makes sense, of course. :). So, yeah, good point, you can keep the http in YourFirstOrchardController.cs and in OwinMiddleware.cs.
But I can see that you changed the URLs in the OrchardHUN.TrainingDemo.csproj too. This means Visual Studio cannot open this project because of the incorrect MSBuild XML namespace.

Comment (MZaXiMuS) [reply Models/PersonListPart.cs]: It was an accident, I noticed that and pushed the original csproj immediately. : )

Comment (domonkosgabor) [reply Models/PersonListPart.cs]: We have 2 csproj files: OrchardHUN.TrainingDemo.csproj and OrchardHUN.TrainingDemo.Tests.csproj.
I'm referring to the first one, which still contains https.

Comment (MZaXiMuS) [reply Models/PersonListPart.cs]: Oh I see, thank you, and sorry.

**Review by domonkosgabor — APPROVED** [review]
